Author: Roberto C. Sanchez <roberto@connexer.com>
Description: Fix transposition of realm and service in debug log (Closes: #732373)
--- cyrus-sasl2.git.orig/saslauthd/cache.c
+++ cyrus-sasl2.git/saslauthd/cache.c
@@ -264,7 +264,7 @@
 		if (memcmp(pwd_digest, read_bucket->pwd_digest, 16) == 0) {
 
 			if (flags & VERBOSE)
-				logger(L_DEBUG, L_FUNC, debug, user, realm, service, "found with valid passwd");
+				logger(L_DEBUG, L_FUNC, debug, user, service, realm, "found with valid passwd");
 
 			cache_un_lock(hash_offset);
 			table_stats->hits++;
@@ -272,14 +272,14 @@
 		}
 
 		if (flags & VERBOSE)
-			logger(L_DEBUG, L_FUNC, debug, user, realm, service, "found with invalid passwd, update pending");
+			logger(L_DEBUG, L_FUNC, debug, user, service, realm, "found with invalid passwd, update pending");
 
 		result->status = CACHE_FLUSH;
 
 	} else {
 
 		if (flags & VERBOSE)
-			logger(L_DEBUG, L_FUNC, debug, user, realm, service, "not found, update pending");
+			logger(L_DEBUG, L_FUNC, debug, user, service, realm, "not found, update pending");
 
 		result->status = CACHE_FLUSH_WITH_RESCAN;
 	}
